src/lm_buddy/jobs/_entrypoints/ragas.py

```diff
@@ -37,13 +36,14 @@ def run_eval(config: RagasJobConfig, artifact_loader: artifact_loader) -> Path:
     ragas_args = {}
 
     # load embedding model
```

Comment:
@imtihan I think there were a few lingering bugs here that I've attempted to fix. The wrong types were being passed to the Langchain classes.